### ISSUE TITLE
Missing wafers in HGCal trigger map

### ISSUE THREAD

**Issue by kpedro88** [cms-sw/cmssw#31506]
#31428 fixed a bug in the definition of valid cells for HGCal.

Since that PR was merged, step2 logs for WF 23234.0 (D49 detector) have a number of messages like this:
```
%MSG-w HGCalTriggerGeometry:  HGCalVFEProducer:hgcalVFEProducer  15-Sep-2020 07:41:38 CEST Run: 1 Event: 1
Found missing wafer (layer=31 u=0 v=1) in trigger modules mapping
%MSG
```

@jbsauvan can you make the necessary updates?

**Comment by kpedro88** [comment]
assign l1,upgrade

**Comment by jbsauvan** [comment]
I will have a look

**Comment by jbsauvan** [comment]
Should be fixed in #31524, with  cms-data/L1Trigger-L1THGCal#20

**Comment by kpedro88** [comment]
+upgrade
